src/main/scala/io/citrine/theta/Stopwatch.scala

```diff
@@ -35,8 +44,14 @@ object Stopwatch {
       val variance = times.map(t => t - mean).map(t => t * t).sum / times.size
 
       /* Estimate the uncertainty in the mean */
-      errorEstimate = Math.sqrt(variance / times.size) / mean
+      errorEstimate = if (times.size > 1) {
+        val dist = new TDistribution(times.size - 1)
+        dist.inverseCumulativeProbability(1.0 - (1.0 - confidence)/2.0) * Math.sqrt(variance / times.size) / mean
```

Comment:
This is enough steps beyond obvious that I would consider putting in a few comments or intermediate variables to spell things out a little more. Up to you.